src/Stratis.Bitcoin.Features.Dns/DnsFeature.cs

```diff
@@ -227,5 +248,14 @@ private void StartWhitelistRefreshLoop()
             this.nodeLifetime.ApplicationStopping,
             repeatEvery: TimeSpan.FromSeconds(30));
         }
+
+        /// <inheritdoc />
+        public override void ValidateDependencies(IFullNodeServiceProvider services)
+        {
+            //TODO: this property should be exposed by FullNodeFeature base class as protected
+            this.services = services;
+
+            services.EnsureServiceIsRegistered<Network>();
```

Comment:
Network is not a service and it will always be registered, not sure we need this.

Reply:
yes was only an example about the usage of EnsureServiceIsRegistered, anyway I'm removing the `services` member now, so this whole method has been removed